spec/takeaway_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:takeaway) { Takeaway.new }
+  let(:menu) { double :menu, see_menu: printed_menu }
+  let(:printed_menu) { "menu" }
```

Comment:
I like the simplicity of using a small string here instead of a whole dummy menu.